src/plugins/drawing/toolbar-buttons/align-button.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useContext } from "react";
+import { observer } from "mobx-react";
+import { TileToolbarButton } from "../../../components/toolbar/tile-toolbar-button";
+import { IToolbarButtonComponentProps } from "../../../components/toolbar/toolbar-button-manager";
+import { DrawingContentModelContext } from "../components/drawing-content-context";
+import { OpenPaletteValues } from "../model/drawing-content";
+import { ToolbarButtonSvg } from "./toolbar-button-svg";
+import { useTouchHold } from "../../../hooks/use-touch-hold";
+import SmallCornerTriangle from "../../../../src/assets/icons/small-corner-triangle.svg";
+import { AlignTypePalette } from "../components/align-palette";
+import { AlignType, getAlignTypeIcon, getAlignTypeTooltip } from "../model/drawing-basic-types";
+
+export const AlignButton = observer(({ name }: IToolbarButtonComponentProps) => {
+  const drawingModel = useContext(DrawingContentModelContext);
+  const isOpen = drawingModel?.openPallette === OpenPaletteValues.Align;
+  const enabled = drawingModel.selection.length > 1;
+
+  const { onClick } = useTouchHold(toggleOpen, alignItems);
+
+  function handleTriangleClick(e: React.MouseEvent) {
+    e.stopPropagation();
+    toggleOpen();
+  }
+
+  function toggleOpen() {
+    if (isOpen) {
+      drawingModel.setOpenPalette(OpenPaletteValues.None);
+    } else {
+      drawingModel.setOpenPalette(OpenPaletteValues.Align);
```

Comment:
After testing out the branch it is a little weird that you have to click the button again after the tool is selected in the open palette to apply the tool.  It seems to me we should immediately apply the alignment as the palette is closed.